client/bootstrap_demo_community.py

```diff
@@ -320,6 +320,53 @@ def test_faucet(client, cid):
         exit(1)
     print('Faucet closed', flush=True)
 
+
+def test_democracy(client, cid):
+    print('Starting democracy test...')
+    client.next_phase()
+    client.next_phase()
+    client.next_phase()
+    #phase is 9, registering
+    print(client.purge_community_ceremony(cid, 1, 8))
+    register_participants_and_perform_meetup(client, cid, accounts)
+    cindex = 9
```

Comment:
that's brittle. we should not assume the same order of tests forever.

Reply:
this is exactly why i have been saying for a long time that we need to refactor the integration tests :)